packages/editor-ui/src/constants.ts

```diff
@@ -141,7 +142,11 @@ export const NON_ACTIVATABLE_TRIGGER_NODE_TYPES = [
 	EXECUTE_WORKFLOW_TRIGGER_NODE_TYPE,
 ];
 
-export const MULTIPLE_OUTPUT_NODE_TYPES = [IF_NODE_TYPE, SWITCH_NODE_TYPE];
+export const MULTIPLE_OUTPUT_NODE_TYPES = [
```

Comment:
can we instead check if that node has multiple outputs? the outputs are specified as an array in the node properties.. can we check for that instead?

Reply:
Great idea! Updated.